### DISCUSSION TITLE
Shortcut for showing/hiding Sidebery?

### DISCUSSION THREAD

**Discussion by al3xandru** [mbnuqw/sidebery#1311] [Q&A]
My apologies if I missed the answer. After going through the settings and the wiki I couldn't figure out if there's a shortcut to show/hide the Sidebery panel. thank you

**Comment by al3xandru** [comment]
Ahh... I have clicked by mistake Ctrl-E and it seems to be doing what I was looking for. Are the shortcuts listed anywhere? For a newbie like me that would be very useful. Thank you!

**Reply by albino1** [reply]
CTRL+E is a built in Firefox shortcut that should select the address bar, so it shouldn't be touching the sidebar.  There's a list of Firefox shortcuts here:

https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly

If you want to see or set any Sidebery shortcuts yourself can just go to Settings -> Keybindings.
